src/component/toolbox/feature/DataView.ts

```diff
@@ -375,7 +377,8 @@ class DataView extends ToolboxFeature<ToolboxDataViewFeatureOption> {
         const blockMetaList = result.meta;
 
         const buttonContainer = document.createElement('div');
-        buttonContainer.style.cssText = 'position:absolute;bottom:5px;left:0;right:0';
+        const buttonExtraCssText = model.get('buttonExtraCssText') || '';
+        buttonContainer.style.cssText = 'position:absolute;' + buttonExtraCssText;
```

Comment:
`buttonExtraCssText` looks like it is for a single button rather than the button container, and can be misleading.

Reply:
How about `buttonContainerExtraCssText`?

Reply:
@plainheart @Ovilia I have renamed it to suggested `buttonContainerExtraCssText`.

Reply:
Any progress on this one ? :)